packages/styles/components/Tooltip.module.scss

```diff
@@ -0,0 +1,104 @@
+@import '~@venice/styles/core';
+
+.tooltip {
+  align-items: center;
+  display: inline-flex;
+  justify-content: center;
+  position: relative;
+
+  .tooltipContent {
+    border: 1px solid #e5e5e5;
+    border-radius: var(--border-radius);
+    display: none;
+    min-width: 150px;
+    padding: 10px var(--space-sm);
+    position: absolute;
+    transition: opacity 0.5s;
+    z-index: 1;
+
+    &::after {
+      border-style: solid;
+      border-width: 5px;
+      content: '';
+      position: absolute;
+    }
+  }
+
+  &:hover {
+    .tooltipContent {
+      display: block;
+      opacity: 1;
+      visibility: visible;
+    }
+  }
+
+  &.dark {
+    .tooltipContent {
+      background-color: var(--default-color);
+      color: var(--white);
+    }
+  }
+
+  &.light {
+    .tooltipContent {
+      background: var(--white);
+      border-radius: 4px;
```

Comment:
We have a variable for it: `var(--border-radius)`

Reply:
Done ✅ 
10d2eb6